CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing to jmap-client-ts
+
+This file was inspired by https://github.com/angular/angular/blob/master/CONTRIBUTING.md
```

Comment:
Good first start, I think this line can be remove. Use also our current jmap-client CONTRIBUTING guide https://github.com/linagora/jmap-client/blob/master/CONTRIBUTING.md ;)

Reply:
I removed this line. I read the file you linked, and I think there is nothing more needed from it.